### PR TITLE
MC-41940: Payment on Account not available in multishipping

### PR DESCRIPTION
**Pull request by thiaramus**
## Purpose of this pull request

This pull request (PR) is to state the limitation of B2B Payment on Account payment method when used in the orders with multiple shipping addresses.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/configuration/sales/payment-methods.html
- https://docs.magento.com/user-guide/payment/payment-on-account.html
- https://docs.magento.com/user-guide/shipping/shipping-multiaddress.html
- https://docs.magento.com/user-guide/configuration/sales/multishipping-settings.html

## Additional information

Confirmed by the product owner:
https://jira.corp.magento.com/browse/MAGETWO-99630?focusedCommentId=1177946&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1177946

### CODE REVIEW

**Comment by thiaramus** [comment]
**A question for reviewer:**

I have placed a callouts about the limitation of the Payment on Account functionality on related pages. Do you think it is worth placing the similar callout on the pages related to order multi-shipping?

Like:
https://docs.magento.com/user-guide/shipping/shipping-multiaddress.html
https://docs.magento.com/user-guide/configuration/sales/multishipping-settings.html

**Comment by dobooth** [comment]
Thanks for asking, @thiaramus. I think it makes sense to add that info box to those pages you referenced.

**Comment by thiaramus** [comment]
> Thanks for asking, @thiaramus. I think it makes sense to add that info box to those pages you referenced.

Thank you, @dobooth. I am adding this info to multi-shipping related pages.

File: src/configuration/sales/multishipping-settings.md [hunk 1]
```diff
@@ -13,3 +13,7 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
 |--- |--- |--- |
 |Allow Shipping to Multiple Addresses|Website|Determines if you allow single orders to be shipping to multiple addresses. Options: Yes / No|
 |Maximum Qty Allowed for Shipping to Multiple Addresses|Website|Sets a limit for the number of units  of a product that can be shipping to multiple addresses.|
+
+{:.bs-callout-info}
+<span class="b2b-only"></span>
```
Comment (thiaramus) [inline src/configuration/sales/multishipping-settings.md:18]: I was struggling to find a way to highlight that this callout is about a B2B related feature. I wasn't able to make `{:.b2b-only}` work next to `{:.bs-callout-info}`, but I found some usages of HTML code like here `src/system/index-management.md:36` that was added relatively recently. Please correct me if there is a better way to do this while keeping it look good :) Thanks.

File: src/configuration/sales/multishipping-settings.md [hunk 2]
```diff
@@ -13,3 +13,6 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
 |--- |--- |--- |
 |Allow Shipping to Multiple Addresses|Website|Determines if you allow single orders to be shipping to multiple addresses. Options: Yes / No|
 |Maximum Qty Allowed for Shipping to Multiple Addresses|Website|Sets a limit for the number of units  of a product that can be shipping to multiple addresses.|
+
+{:.bs-callout-info}
+<span class="b2b-only"></span> For orders with multiple shipping addresses, [Payment on Account]({% link payment/payment-on-account.md %}) payment method, even if enabled, will not be available during the checkout.
```
Comment (thiaramus) [inline src/configuration/sales/multishipping-settings.md:18]: I was struggling to find a way to highlight that this callout is about a B2B related feature. I wasn't able to make `{:.b2b-only}` work next to `{:.bs-callout-info}`, but I found some usages of HTML code like here `src/system/index-management.md:36` that was added relatively recently. Please correct me if there is a better way to do this while keeping it look good :) Thanks.

File: src/shipping/shipping-multiaddress.md [hunk 3]
```diff
@@ -20,3 +20,6 @@ The Multiaddress Shipping options enable customers to ship an order to multiple
 
    ![]({% link images/images/config-sales-multishipping-settings-options.png %}){: .zoom}
    _[Multiaddress Shipping Options]({% link configuration/sales/multishipping-settings.md %})_
+
+{:.bs-callout-info}
+<span class="b2b-only"></span> For orders with multiple shipping addresses, [Payment on Account]({% link payment/payment-on-account.md %}) payment method, even if enabled, will not be available during the checkout.
```
Comment (thiaramus) [inline src/shipping/shipping-multiaddress.md:25]: Same thing I mentioned in the previous comment. Please correct me if there is a better way to specify a B2B related feature in callout.

File: src/configuration/sales/multishipping-settings.md (hunk 2 above)
Comment (dobooth) [reply src/configuration/sales/multishipping-settings.md:18]: That's great. Can't argue with precedent.

Comment (dobooth) [inline src/configuration/sales/multishipping-settings.md:18]:
```suggestion
<span class="b2b-only"></span> For orders with multiple shipping addresses, the [Payment on Account]({% link payment/payment-on-account.md %}) payment method, even if enabled, will not be available during the checkout.
```

File: src/shipping/shipping-multiaddress.md (hunk 3 above)
Comment (dobooth) [inline src/shipping/shipping-multiaddress.md:25]:
```suggestion
<span class="b2b-only"></span> For orders with multiple shipping addresses, the [Payment on Account]({% link payment/payment-on-account.md %}) payment method, even if enabled, will not be available during the checkout.
```

**Review by dobooth — CHANGES_REQUESTED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1207`.

**Pull request merged**